incubator/elasticsearch/templates/coordinating-deploy.yaml

```diff
@@ -1,34 +1,34 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: "{{ template "client.fullname" . }}"
+  name: "{{ template "coordinating.fullname" . }}"
   labels:
     app: "{{ template "name" . }}"
```

Comment:
As per the new standards following the upstream charts repo, this template should be named `elasticsearch.name`. ref https://github.com/bitnami/charts/blob/master/incubator/node/templates/_helpers.tpl#L5-L7